frontend/src/static/css/style.scss

```diff
@@ -29,7 +29,7 @@ a {
 
 .loader {
   align-items: center;
-  background: #FFF;
+  background: mui-color('white');
```

Comment:
irrelevant to this PR, and should be removed.

Reply:
I agree that it is not specific to this PR but this was such a minute change that I didn't wish to create a new PR for this. So are you suggesting: 
1) This isn't an issue and doesn't need to be fixed
2) Do a different PR for fixing this 
3) Include this in a PR which affects css styles

Reply:
Yes, `This isn't an issue and doesn't need to be fixed`, or at least not necessary unless we make direct changes to the file.